linux/hid.c

```diff
@@ -1114,13 +1115,21 @@ int HID_API_EXPORT_CALL hid_get_serial_number_string(hid_device *dev, wchar_t *s
 	return get_device_string(dev, DEVICE_STRING_SERIAL, string, maxlen);
 }
 
-int HID_API_EXPORT_CALL hid_get_indexed_string(hid_device *dev, int string_index, wchar_t *string, size_t maxlen)
+int HID_API_EXPORT_CALL hid_get_indexed_string(hid_device * dev, int string_index, wchar_t * string, size_t maxlen)
```

Comment:
I understand that we don't have a strict code-style document,
and we won't debate about the "west-const vs east-const" holy-war here.

But a **consistency** across implementation is a must-have - please revert this line.